src/UI/Component/ViewControl/Factory.php

```diff
@@ -94,4 +94,30 @@ public function section(Button $previous_action, \ILIAS\UI\Component\Component $
 	 */
 	public function sortation(array $options);
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      Data with a larger amount of entries might be displayed in parts
+	 *      by limiting the number of visible entries for a view.
+	 *      The pagination enables the user to select a range of entries.
+	 *      This control applies to all sorts of _structured_ data, like tables and lists.
+	 *   composition: >
+	 *      Pagination is a collection of shy-buttons.
```

Comment:
There is more to that. One of those buttons is always active right? The ones on the left and right look special, what are those? What about the pagination of empty lists? Is the Pagination also empty?